<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>570</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="p_groupBox_SendData">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Send data</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_14" stretch="0,0,0">
       <item>
        <widget class="QGroupBox" name="p_groupBox_ChangingEnterField">
         <property name="title">
          <string>Switch the enter method:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="p_radioButton_ASCII_Text">
            <property name="text">
             <string>ASCII text</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="p_radioButton_HexTable">
            <property name="text">
             <string>Hex table</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="p_groupBox_EnterCommand">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Enter the command:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLineEdit" name="p_lineEdit_SendLine">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="p_tableView_SendTable">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="p_pushButton_Send">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Send</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="p_groupBox_History">
      <property name="title">
       <string>History</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QTabWidget" name="p_tabWidget_History">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab_table_History_hex">
          <attribute name="title">
           <string>Table Hex</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QTableView" name="p_tableView_History_hex"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_table_History_ASCII">
          <attribute name="title">
           <string>Table ASCII</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="QTableView" name="p_tableView_History_ASCII"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="p_groupBox_SendedData">
      <property name="title">
       <string>Sended data</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <widget class="QTabWidget" name="p_tabWidget_SendedView">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab_table_Sended_hex">
          <attribute name="title">
           <string>Table Hex</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QTableView" name="p_tableView_SendedData_hex"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_table_Sended_ASCII">
          <attribute name="title">
           <string>Table ASCII</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QTableView" name="p_tableView_SendedData_ASCII"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="p_groupBox_ReceivedData">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Received data</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QTabWidget" name="p_tabWidget_ReceivedView">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab_table_Received_hex">
          <attribute name="title">
           <string>Table Hex</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTableView" name="p_tableView_ReceivedData_hex"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_table_Received_ASCII">
          <attribute name="title">
           <string>Table ASCII</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTableView" name="p_tableView_ReceivedData_ASCII"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Connection">
    <property name="title">
     <string>&amp;Connection</string>
    </property>
    <addaction name="action_New_connection"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_Connection"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_New_connection">
   <property name="text">
    <string>Connection &amp;Settings</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="action_Save_log">
   <property name="text">
    <string>&amp;Save log</string>
   </property>
  </action>
  <action name="actionSave_log_as">
   <property name="text">
    <string>Save log &amp;as...</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>p_radioButton_ASCII_Text</sender>
   <signal>toggled(bool)</signal>
   <receiver>p_lineEdit_SendLine</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>p_radioButton_HexTable</sender>
   <signal>toggled(bool)</signal>
   <receiver>p_tableView_SendTable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
